docs/releases/1.11.txt

```diff
@@ -403,3 +403,12 @@ Miscellaneous
   :class:`~django.contrib.auth.views.PasswordResetDoneView`,
   :class:`~django.contrib.auth.views.PasswordResetConfirmView`, and
   :class:`~django.contrib.auth.views.PasswordResetCompleteView`.
+
+* The new :func:`~django.test.utils.teardown_databases` makes it easier to
```

Comment:
This should be more explicit about what's deprecated. Also a note in `docs/internals/deprecation.txt` is missing.